packages/server/tests/acceptance/estimateGasPrecompile.spec.ts

```diff
@@ -262,13 +309,181 @@ describe('EstimatePrecompileContract tests', function () {
     );
   };
 
+  const isEqualWithDeviation = (actualGasUsed: bigint, expectedGasUsed: number, percentDeviation: number) => {
```

Comment:
PR #2033 Adds the configurable property TEST_GAS_PRICE_DEVIATION.  Perhaps that can be used here as well to be consistent.